### PR TITLE
fixes #84

### LINKED ISSUE #84: Document how to manually create trust policy for RC1 first run experience
For RC1 we don't have good experience to create trust policy even for the test scenario `notation cert generate-test`. We should document the manual steps to create policy for the test cert and place it in the correct place depending on the OS folder structure.

CC:// @zr-msft

### PR DESCRIPTION
**Pull request by zr-msft**
Signed-off-by: Zach Rhoads <zachary.rhoads@microsoft.com>

### CODE REVIEW

File: content/en/docs/quickstart.md [hunk 1]
```diff
@@ -71,13 +71,15 @@ Use `notation sign` to sign the container image.
 notation sign $IMAGE
 ```
 
-By default, the signature format is `JWS`. Use `--signature-format` to use `COSE` signature format.
+By default, the signature format is `JWS`. Use `--signature-format` to use [COSE](https://cose-wg.github.io/cose-spec/) signature format.
```
Comment (FeynmanZhou) [inline content/en/docs/quickstart.md:74]: Let's use https://datatracker.ietf.org/doc/html/rfc8152 for COSE's hyperlink.

File: content/en/docs/quickstart.md [hunk 2]
```diff
@@ -145,32 +143,36 @@ notation verify $IMAGE
...
 docker rm -f $(docker ps -q)
 ```
 
-## What's Next
+## Next steps
+
+For additional examples on using Notary, see:
 
-Notation can do much more than what is discussed in the quick start. Learn more information from other sections of this site.
+- [Build, sign, and verify container images using Notary and Azure Key Vault](https://learn.microsoft.com/azure/container-registry/container-registry-tutorial-sign-build-push)
```
[the remark this answers is not in the document]
Comment (toddysm) [reply content/en/docs/quickstart.md:178]: Agree, we should remove the cloud-specific details from the generic guides. We should have a separate page for integrations like Azure Key Vault, AWS KMS, etc.

File: content/en/docs/quickstart.md (hunk 1 above)
Comment (zr-msft) [reply content/en/docs/quickstart.md:74]: done

File: content/en/docs/quickstart.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (zr-msft) [reply content/en/docs/quickstart.md:178]: removing what's next section until we have more content to link to

**Comment by zr-msft** [comment]
@FeynmanZhou and @toddysm incorporated feedback

**Comment by zr-msft** [comment]
also, created this issue for a net-new doc

https://github.com/notaryproject/notaryproject.dev/issues/97

**Review by FeynmanZhou — APPROVED** [review]
LGTM

**Comment by FeynmanZhou** [comment]
@yizha1 Do you have any comments on this PR?

File: content/en/docs/quickstart.md [hunk 3]
```diff
@@ -145,32 +143,30 @@ notation verify $IMAGE
...
+### Remove the notation configuration
 
-* Remove local keys, self-signed certificates and notation configurations
+To remove local keys, self-signed certificates, and notation configurations, remove the notation configuration directory.
   
-For linux user,
+For linux:
 
 ```console
 rm -r $HOME/.config/notation/
```
Comment (yizha1) [inline content/en/docs/quickstart.md:155]: {$HOME}/.config/notation/

File: content/en/docs/quickstart.md [hunk 4]
```diff
@@ -145,32 +143,30 @@ notation verify $IMAGE
...
 ```
 
-For a Mac user
+For macOS:
 
 ```console
 rm -r $HOME/Library/Application Support/notation/
```
Comment (yizha1) [inline content/en/docs/quickstart.md:161]: ${HOME}/Library/Application Support/notation/

File: content/en/docs/quickstart.md [hunk 5]
```diff
@@ -145,32 +143,30 @@ notation verify $IMAGE
...
 ```
 
-For a Window user, delete the directory `C:\Users\<username>\AppData\Roaming\notation\`
+For Windows, delete the directory `C:\Users\<username>\AppData\Roaming\notation\`
```
Comment (yizha1) [inline content/en/docs/quickstart.md:164]: %USERPROFILE%\AppData\Roaming\notation\

**Review by yizha1 — APPROVED** [review]
LGTM with comments. We can improve it step by step.

**Pull request merged**